scripts/turtles-quickstart.sh

```diff
@@ -0,0 +1,297 @@
+  image: kindest/node:v1.30.0
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    protocol: TCP
+  extraMounts:
+  - hostPath: /var/run/docker.sock
+    containerPath: /var/run/docker.sock
+EOF
+
+kind create cluster --config "$BASEDIR/kind-config.yaml" --name $RANCHER_CLUSTER_NAME
+
+docker pull $RANCHER_IMAGE
+kind load docker-image $RANCHER_IMAGE --name $RANCHER_CLUSTER_NAME
+
+kubectl rollout status deployment coredns -n kube-system --timeout=90s
+
+kubectl cluster-info --context kind-$RANCHER_CLUSTER_NAME
+kubectl wait --for=condition=Ready node/$RANCHER_CLUSTER_NAME-control-plane
+kubectl get nodes -o wide
+
+show_step "Install Ingress NGINX controller on ${RANCHER_CLUSTER_NAME}"
+
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+helm repo update
+helm install --create-namespace --namespace ingress-nginx ingress-nginx ingress-nginx/ingress-nginx --set controller.hostNetwork=true
+
+show_step "Install Cert Manager on ${RANCHER_CLUSTER_NAME}"
+
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install cert-manager jetstack/cert-manager \
+    --namespace cert-manager \
+    --create-namespace \
+    --set crds.enabled=true
+
+show_step "Install Rancher latest version on ${RANCHER_CLUSTER_NAME}"
+
+helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+helm repo update
+helm install rancher rancher-latest/rancher \
+    --namespace cattle-system \
+    --create-namespace \
+    --set bootstrapPassword="$RANCHER_PASSWORD" \
+    --set replicas=1 \
+    --set hostname="$RANCHER_HOSTNAME" \
+    --set ingress.ingressClassName=nginx \
+    --version="$RANCHER_VERSION" \
+    --wait
+
+kubectl rollout status deployment rancher -n cattle-system --timeout=180s
+
+cat > rancher-settings.yaml <<EOF
+apiVersion: management.cattle.io/v3
+kind: Setting
+metadata:
+  name: server-url
+value: https://${RANCHER_HOSTNAME}
+---
+apiVersion: management.cattle.io/v3
+kind: Setting
+metadata:
+  name: first-login
+value: "false"
+---
+apiVersion: management.cattle.io/v3
+kind: Setting
+metadata:
+  name: eula-agreed
+value: "2023-08-16T20:39:11.062Z"
+---
+apiVersion: management.cattle.io/v3
+kind: Setting
+metadata:
+  name: agent-tls-mode
+value: "system-store"
+EOF
+
+kubectl apply -f rancher-settings.yaml
+
+kubectl rollout status deployment rancher -n cattle-system --timeout=180s
+
+show_step "Waiting for Rancher to be READY ..."
+
+until [[ $(curl -ks https://${RANCHER_HOSTNAME}/ping) == "pong" ]]; do
+    sleep 1
+done
+echo "Received pong!"
+
+show_step "Install Rancher Turtles on ${RANCHER_CLUSTER_NAME}"
+
+helm repo add turtles https://rancher.github.io/turtles
+helm repo update
+helm install rancher-turtles turtles/rancher-turtles --version v0.16.0 \
```

Comment:
should the Turtles version be configurable?

Reply:
done